package.json

```diff
@@ -1,14 +1,15 @@
 {
-    "name": "OpenSlides",
-    "private": true,
-    "devDependencies": {
-        "bower": "~1.3.12",
-        "gulp": "~3.8.10",
-        "gulp-concat": "~2.4.3",
-        "gulp-if": "~1.2.5",
-        "gulp-minify-css": "~0.3.11",
-        "gulp-uglify": "~1.0.2",
-        "main-bower-files": "~2.4.1",
-        "yargs": "~1.3.3"
-    }
+  "name": "OpenSlides",
+  "private": true,
+  "devDependencies": {
+    "bower": "~1.3.12",
+    "gulp": "~3.8.10",
+    "gulp-concat": "~2.4.3",
+    "gulp-if": "~1.2.5",
+    "gulp-minify-css": "~0.3.11",
+    "gulp-uglify": "~1.0.2",
+    "main-bower-files": "~2.4.1",
+    "yargs": "~1.3.3",
+    "po2json": "~0.3.2"
```

Comment:
The converted json files lie in openslides/static and so there are not part of the repo. You can create a gulp job for it if you like. It could look for files "openslides/locale/[^(en)]/javascript.po" and generates files "openslides/static/translation/$1.json".

At the moment po2json is not compatible with the current jed version, see https://github.com/SlexAxton/Jed/issues/36

I would propose we wait until we have some translated po files and look if there is a newer version of po2json, or use jed ~1.0.0